.travis.yml

```diff
@@ -6,15 +6,10 @@ script:
 matrix:
   include:
     - rvm: "1.9.3"
-      gemfile: Gemfile-old-ruby
     - rvm: "2.0.0"
-      gemfile: Gemfile-old-ruby
     - rvm: "2.1.0"
-      gemfile: Gemfile-old-ruby
     - rvm: "2.1.5"
-      gemfile: Gemfile-old-ruby
     - rvm: "2.2.1"
-      gemfile: Gemfile-old-ruby
     - rvm: "2.2.2"
     - rvm: "2.3.0"
```

Comment:
How about dropping the support for ruby < 2.4, because they are in EOL

Reply:
Since it was "easy" to check support for ruby >= 2.3, I've left until 2.3, but removed older ones